mteb/tasks/Audio/AudioClassification/eng/CommonLanguageAgeDetection.py

```diff
@@ -0,0 +1,62 @@
+from __future__ import annotations
+
+from mteb.abstasks.Audio.AbsTaskAudioClassification import (
+    AbsTaskAudioClassification,
+)
+from mteb.abstasks.TaskMetadata import TaskMetadata
+
+
+class CommonLanguageAgeDetection(AbsTaskAudioClassification):
+    metadata = TaskMetadata(
+        name="Common-Language-Age-Detection",
```

Comment:
Could we follow camel case here?
```suggestion
        name="CommonLanguageAgeDetection",
```